### PR TITLE
Move sound effect out of event loop. convert snap.wav from 32bit floa… [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by vladest**
…t to 16 bit to be supported by QSound

Test plan:
- Make sure notification sounds turned on
- Open Tablet, press Snap
- Hear snapshot sound

### CODE REVIEW

**Comment by ZappoMan** [comment]
hmmmm - something about this PR just made me realize... the old code (and this new code) assumes that Qt is going to be playing to the same audio device as we're playing to... it's possible that those could be out of sync and users won't hear the sound effect. @zzmp @howard-stearns FYI

**Comment by hifi-gustavo** [comment]
Build console output for commit: a607d7692a078048524936d4b20f6464fb6bc544 available at: https://gist.github.com/f94284a10fa942c0da16cd83a316a75a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10194-a607d76.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10194-a607d76.dmg

**Comment by claudioshg** [comment]
testing

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift and in Desktop Mode

Tested: Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10194-a607d76.exe

- Make sure notification sounds turned on: Success
- Open Tablet, press Snap: Success
- Hear snapshot sound: Success!

PASS

**Comment by howard-stearns** [comment]
I'm sorry that I didn't follow up on this. I think @ZappoMan is right.  Don't we want to make this into a local injector rather than a QSound, so that it goes through the Interface-selected device?
Could you look into that @vladest ?

**Comment by vladest** [comment]
@howard-stearns I'll take a look

**Comment by vladest** [comment]
It seems. if we want follow selected audio device for notifications, need to implement notifications sounds thru AudioClient

**Comment by vladest** [comment]
I suppose that the PR might be used as it is right now, since the issue is fixed and previous code used system default variant instead of selected one. Injecting notification sounds to existing audio processing code will be comparably complex, so it might be good to create another issue for this

**Comment by zzmp** [comment]
I agree with @valdest. The original intent was to move the sound out of the event loop - that is fixed.
A new issue was discovered - that the sound is played through Qt, and not through the AudioClient - and that should be fixed in a separate PR.

File: interface/src/Application.h [hunk 1]
```diff
@@ -686,6 +687,7 @@ private slots:
     QTimer _addAssetToWorldErrorTimer;
 
     FileScriptingInterface* _fileDownload;
+    QSound *_snapshotSound;
```
Comment (sethalves) [inline interface/src/Application.h]: make this "QSound* " rather than "QSound *" please (space after the *)

**Comment by anshuman64** [comment]
@vladest After talking to Howard, I'm making a call: let's get this PR merged. I have created a new issue to move the sound to AudioClient: https://github.com/highfidelity/hifi/issues/10237

**Comment by hifi-gustavo** [comment]
Build console output for commit: 901ddfda5e68a6d7ca9f303f554fe71d048d7134 available at: https://gist.github.com/f94284a10fa942c0da16cd83a316a75a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10194-901ddfd.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10194-901ddfd.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 055c62d18f5c93e9ec6845ca7cb25c1d4f7eabc6 available at: https://gist.github.com/f94284a10fa942c0da16cd83a316a75a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10194-055c62d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10194-055c62d.dmg

**Comment by vladest** [comment]
Obsoleted due to https://github.com/highfidelity/hifi/pull/10248

**Comment by zzmp** [comment]
@vladest can this be closed, then?

**Comment by vladest** [comment]
yes

**Pull request closed without merging**